### PR TITLE
Attract Helpers

### PR DESCRIPTION
**Pull request by AryaBuddha**
Hey Kalle!

Just wanted to give you some tips on open source projects. Here is an excellent example of an open-source project https://github.com/crybapp . Some ways you can get people helping out with your document is to add a tag named "good first issue". That would bring new people in to help out and more experienced people in to help out with the harder issues.

I updated the readme to include a "want to help out" section and removed a bullet point from the description.

### CODE REVIEW

**Comment by TNeri1** [comment]
I like your addition to attract new comers. I am still in school, but I have learned C++ (Just finished up OOP, now getting on to Data Structures), SQL, R, JavaScript, HTML, CSS, and now learning Python. I hope I can be able to contribute in some small way or another. I want to help!

**Comment by AryaBuddha** [comment]
@TNeri1 That's exactly my goal! I want to administer to as many newcomers as possible to help the project ans well as the individual to learn more about the topic.

File: README.md [hunk 1]
```diff
@@ -32,3 +32,10 @@
 ### Edge cases:
 - User1 is in the process of booking an appointment and User2 books it before User1 finishes. So once User1 is done it is no longer available.
     - Solution: When a user presses a time slot it becomes unavailable straight away. And if they cancel or get disapproved it will become available again.
+
+
+---
+
+## Want to help out?
+I encourage everyone to help out with this project, no matter what your skill base is. If you know any form of coding, look at the issues (in special those labelled as "good first issue", for the newcomers), and make Pull Requests! Make issues! Make reviews! Make!
```
Comment (robsmithw) [inline README.md:56]: I like what you're going for but the last part seems a bit wordy.

**Review by robsmithw — CHANGES_REQUESTED** [review]

**Comment by AryaBuddha** [comment]
This seems like a lost cause, thanks guys
